src/statement.ts

```diff
@@ -117,15 +128,15 @@ export class Statement<S = unknown> {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async all<T = S, K extends keyof T = keyof T>(
     colName?: undefined,
-    opts?: Signal
+    opts?: SignalAndInterval
   ): Promise<Result<T>>;
   async all<T = S, K extends keyof T = keyof T>(
     colName: K,
-    opts?: Signal
+    opts?: SignalAndInterval
   ): Promise<Result<T[K]>>;
   async all<T = S, K extends keyof T = keyof T>(
     colName?: K,
-    opts: Signal = {}
+    opts: SignalAndInterval = {}
```

Comment:
I think we need to allow passing an interval to the Statement methods in order to enable setting the value to something other than the default.